src/controllers/users.controller.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+/* eslint no-console: ["error", { allow: ["warn", "error"] }] */
+
+const usersService = require('../services/user.service');
+
+const getAll = async(req, res) => {
+  try {
+    const result = await usersService.getAllUsers();
+
+    res.send(result.map(user => usersService.normalizeUserData(user)));
+  } catch (err) {
+    console.error('Error executing query:', err);
+
+    res.status(500).send(err.message);
+  }
+};
+
+const getParticular = async(req, res) => {
```

Comment:
getUser or getUserById would be much appropriate name of the function
```suggestion
const getUser = async(req, res) => {
```